lib/recommended-rules.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
   "ember/no-attrs-snapshot": "error",
   "ember/no-capital-letters-in-routes": "error",
   "ember/no-duplicate-dependent-keys": "error",
+  "ember/no-ember-super-in-es-classes": "error",
```

Comment:
In general, we do not update recommended rule set as part of adding a new rule. This is because its considered a breaking change. 

Is the idea here that using `this._super` is _fundamentally_ broken already, and adding this rule to the recommended/default set is is therefore not a breaking change?

Reply:
That was my reasoning going into it, but on reflection it would still be pretty annoying to have CI start failing on a project because of this, so I'll back that out.